packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1224,6 +1224,47 @@ class EditableText extends StatefulWidget {
   ///   works only with [TextInputType.emailAddress]. Make sure the input field has a
   ///   compatible [keyboardType]. Empirically, [TextInputType.name] works well
   ///   with many autofill hints that are predefined on iOS.
+  ///
+  /// ### Troubleshooting Autofill
+  ///
+  /// Platform autofill services rely on [autofillHints] to work. Make sure the
+  /// entries in [autofillHints] are supported by the target platform, and the
+  /// autofill service currently in use (the name of the service can typically
+  /// be found in your mobile device's system settings).
+  ///
+  /// #### Autofill UI refuses to show up when I tap on the text field?
+  ///
+  /// Check if autofill is turned on on the target platform. Also make sure
+  /// there're available credentials stored in the autofill service.
+  ///
+  /// - iOS password autofill: Go to Settings -> Password, turn on "Autofill
```

Comment:
uber-nit: we usually use "*" over "-" for lists. also, the next lines after the first one of a bullet point should be indented by two spaces.

Reply:
changed to lower case, but kept some "Autofill" because those are verbatim.